static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx

```diff
@@ -90,6 +93,7 @@ export function TimelineTableRow({
       ? [
           (env: string, isMuted: boolean) => ({
             label: isMuted ? t('Unmute Environment') : t('Mute Environment'),
+            details: monitor.isMuted ? t('Monitor is muted') : undefined,
             key: 'mute',
             onAction: () => onToggleMuteEnvironment(env, !isMuted),
           }),
```

Comment:
`DropdownMenu` supports a `disabled` key here, which when true makes there label text gray and disabled pointer events.

Reply:
ahh excellent will use that to disable this option too